pkg/api/component_report.go

```diff
@@ -472,6 +477,29 @@ func (c *componentReportGenerator) getJobRunTestStatusFromBigQuery() (apitype.Co
 }
 
 func (c *componentReportGenerator) getCommonTestStatusQuery() (string, string, []bigquery.QueryParameter) {
+
+	// Simulate a list of varaints that were requested, this would be specified in the API request / UI in the real world...
+	variants := []string{"Network", "Upgrade", "Architecture", "Platform"}
```

Comment:
groupBy query param becomes the above array

Additional includeVariant query param comes in with Installer:hypershift for example. When includes are specified we inject a join and a where, but not a groupby for that variant name.